doc/howto/network_ovn_setup.md

```diff
@@ -21,7 +21,7 @@ Create an OVN network and an instance using it:
 ```
 lxc network set lxdbr0 ipv4.dhcp.ranges=... ipv4.ovn.ranges=... # Allocate IP range for OVN gateways.
 lxc network create ovntest --type=ovn network=lxdbr0
-lxc init images:ubuntu/20.04 c1
+lxc init images:ubuntu/22.04 c1
```

Comment:
Should this be ubuntu:22.04?

Reply:
I would think the community image would work here. If you prefer the official one, it's OK with me.

Reply:
It looks like we are standardizing on the `ubuntu:...` image most other places.